utils/docker/Dockerfile.centos.7

```diff
@@ -119,6 +121,7 @@ RUN dnf -y install deltarpm && \
         patch \
         patchelf \
         pciutils \
+        python-pyelftools \
```

Comment:
Let's not forget to remove this if it is indeed a python3 package we want and not a python2 package, just to avoid unnecessary bloat in this file.